lib/views/github-status-bar-tile.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Octicon from '../atom/octicon';
+
+import {addEvent} from '../reporter-proxy';
+import {autobind} from '../helpers';
+
+export default class GithubStatusBarTile extends React.Component {
+  static propTypes = {
+    didClick: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    autobind(this, 'handleClick');
+  }
+
+  handleClick() {
+    addEvent('click', {package: 'github', component: 'GithubStatusBarTile'});
```

Comment:
Not sure what convention we're going with for event names (or if we have even discussed one yet)... I had the thought that a more descriptive event name could be handy, something like `toggle-github-tab`. That way whoever is combing through the data we're collecting can more easily infer what the events are for in terms of extracting information about user behavior.

Reply:
That said, I can also see the value in simply stating that we're capturing a click event on a particular view element... 

Oh okay, so we want to decouple the UI action (click on an element) from the effect of the action (toggling the tab). We've instrumented the `reveal` and `hide` methods of the `TabTracker` which tells us the number of times a given tab is opened/closed. We also want to know what method was used to reveal/hide the tab, which is what we're doing here. Does that sounds about right?